### ISSUE TITLE
The fine tune can't work

### ISSUE THREAD

**Issue by youth123** [Lyken17/pytorch-memonger#5]
Thanks for sharing the tools. It helps me a lot. But I have some little question. When I load the model parameters trained before to continue my traininig, the accuracy can not improve after many epoches. I thought this is because the batch normalization parameters is  sqrted when saved.

**Comment by Lyken17** [comment]
Sorry for the late rely. Do you still have the issue? If so, please attach a minimal case.
